runtime/src/tokio/runtime.rs

```diff
@@ -32,6 +32,8 @@ use std::{
 };
 use tokio::runtime::{Builder, Runtime};
 
+const TICK_INTERVAL: Duration = Duration::from_secs(10);
```

Comment:
```suggestion
const PROCESS_METRICS_TICK_INTERVAL: Duration = Duration::from_secs(10);
```
Doesn't hurt to be a bit more verbose in a busy file like this one. (Speaking of which, we could probably stand to split some of this file out into separate files.)